<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutQtadsDialog</class>
 <widget class="QDialog" name="AboutQtadsDialog">
  <property name="windowTitle">
   <string>About QTads</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="aboutLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Contributors</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="plainText">
          <string>(All entries are listed alphabetically by first name.)

Thanks fly out to:

  Adam Thornton
  Adrien Beau
  Alan Mead
  Andrew Plotkin
  Beni Serfaty
  C.E.J. Pacian
  Chris Odhner
  Christophe Antoine
  Curt Siffert
  Daniel Schepler
  Delta477
  Denis Pokataev
  Dominik 'disconnect3d' Czarnota
  Emily Boegheim
  Eric Eve
  Eric Forgeot
  Erik Temple
  František Dvořák
  Greg Boettcher
  Guilherme De Sousa
  Hugo Landau
  Jim Aikin
  Jim Nelson
  Joe Mason
  John Ziegler
  Kevin Boone
  Kodath duMatri
  Kris Kennaway
  Krister Fundin
  Leif Huhn
  Marc Simpson
  Ming Hua
  NR Turner
  Nicolas Lécureuil
  Ornithopter
  Petter Sjölund
  Poster
  Rune Berg
  Rémi Verschelde
  Samuel Verschelde
  Stefan Müller
  Thom Brown
  Tomas Blaha
  Tony Houlbrooke
  Xenius
  tetzank

for the ideas, patches and bug reports. Also thanks to everyone who donated to this project.

Special thanks to Mike Roberts for creating TADS and for providing help and explaining the inner workings of TADS.</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextBrowserInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Version</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="versionInfoLabel">
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="indent">
          <number>20</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextBrowserInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AboutQtadsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AboutQtadsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
